pulsar-broker/src/test/java/org/apache/pulsar/broker/namespace/NamespaceServiceTest.java

```diff
@@ -602,8 +602,8 @@ public void testSplitLargestBundle() throws Exception {
      */
     @Test
     public void testSplitBundleWithHighestThroughput() throws Exception {
-
         conf.setLoadManagerClassName(ModularLoadManagerImpl.class.getName());
+        conf.setLoadBalancerEnabled(false);
```

Comment:
Hmmm, it's a little confusing. The test is related to the load manager but need to disable the load balancer in the test.

Reply:
I think without this change, the flaky test can also be fixed? Since we will not return null for method `getBundleWithHighestThroughput`

Reply:
No, disable the load balancer just to cancel the load report task. 

You can see the below code, in this test, it tries to write the mock load data to storage, but if the load report task starts, the mock data might be overridden.

https://github.com/apache/pulsar/blob/01349088f86ad8034c844fb1bbe0b0c04e2c0892/pulsar-broker/src/test/java/org/apache/pulsar/broker/namespace/NamespaceServiceTest.java#L625-L634